### PR TITLE
Exploration society

### PR DESCRIPTION
**Pull request by starfox-ac**
- Add Exploration Society weenies from The Risen Princess patch: three weapon gems, 12 stamped letters, 14 weapon casts, Amateur and Seasoned Explorer equipment
- Add recipes for crafting the Amateur and Seasoned Explorer melee weapons from the Weapon Gems and Casts
- Add reward turn-ins to Society Agents
- Remove incorrectly assigned DR on TOD patch folder mobs
- Add missing Repugnant Eater kill count quest
- Fix two miss-labeled patch folders
- Add Hoary Mattekar Over-robe and add emote to Gillian the Poacher and Yi Yo-Jin to exchange for Hoary Mattekar Robe
- Update Aun Autuorea per LSD and add new required quest entry

### CODE REVIEW

**Comment by LtRipley36706** [comment]
Is this ready for review?

**Comment by starfox-ac** [comment]
yes

File: Database/Patches/2007-07-IntelligentDesigns/9 WeenieDefaults/Gem/35539 _Club of Undead Bashing_ Summoning Gem.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+VALUES (35539,   1,   33556223) /* Setup */
+     , (35539,   3,  536870932) /* SoundTable */
+     , (35539,   6,   67111928) /* PaletteBase */
+     , (35539,   7,  268435841) /* ClothingBase */
+     , (35539,   8,  100673039) /* Icon */
+     , (35539,  22,  872415275) /* PhysicsEffectTable */
+     , (35539,  50,  100673774) /* IconOverlay */
+     , (35539,  52,  100689403) /* IconUnderlay */;
+
+INSERT INTO `weenie_properties_emote` (`object_Id`, `category`, `probability`, `weenie_Class_Id`, `style`, `substyle`, `quest`, `vendor_Type`, `min_Health`, `max_Health`)
```
Comment (LtRipley36706) [inline Database/Patches/2007-07-IntelligentDesigns/9 WeenieDefaults/Gem/35539 _Club of Undead Bashing_ Summoning Gem.sql]: Swap emote/actions for a couple of properties
see [Boxed Aug Gem](https://github.com/ACEmulator/ACE-World-16PY-Patches/blob/master/Database/Patches/2010-08-TheRisenPrincess/9%20WeenieDefaults/Gem/Misc/46441%20Boxed%20Augmentation%20Gem.sql) for example

File: Database/Patches/2010-08-TheRisenPrincess/9 WeenieDefaults/Book/46283 Pragmatic Guide to Atlan Weapons.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+VALUES (46283,   1,   33554771) /* Setup */
+     , (46283,   3,  536870932) /* SoundTable */
+     , (46283,   8,  100668117) /* Icon */
+     , (46283,  22,  872415275) /* PhysicsEffectTable */;
+
+INSERT INTO `weenie_properties_book` (`object_Id`, `max_Num_Pages`, `max_Num_Chars_Per_Page`)
+VALUES (46283, 4, 1000);
+
+INSERT INTO `weenie_properties_book_page_data` (`object_Id`, `page_Id`, `author_Id`, `author_Name`, `author_Account`, `ignore_Author`, `page_Text`)
+VALUES (46283, 0, 4294967295, 'Explorer Scribe', 'Password is cheese', False, 'Atlan Weapons\n\nAtlan weapons have become sought after once again. These ancient weapons grant great power to those who wield them, but in order to wield them, one must first construct them.')
```
Comment (LtRipley36706) [inline Database/Patches/2010-08-TheRisenPrincess/9 WeenieDefaults/Book/46283 Pragmatic Guide to Atlan Weapons.sql]: prewritten

**Pull request merged**